charmcraft.yaml

```
@@ -140,6 +140,10 @@ config:
         This value should be equal to or shorter than half the root-ca-validity.
         Changing this value will trigger generation of a new CA certificate,
         revoking all previously issued certificates.
+    certificate-number-limit:
```

```suggestion
    certificate-limit:
```

The `number` is obvious since the type is an int

---

Done